web/viewer.css

```diff
@@ -920,6 +920,24 @@ html[dir="rtl"] #viewOutline.toolbarButton::before {
   content: url(images/toolbarButton-search.png);
 }
 
+.toolbarButton.pdfSidebarNotification::after {
+  position: absolute;
+  display: inline-block;
+  top: 0px;
+  /* Create a filled circle, with a diameter of 10 pixels, using only CSS: */
+  content: '';
+  background-color: #8BE651;
+  height: 10px;
```

Comment:
> The visual notification is clear enough to show the user that attachments/outlines exist,

Hopefully you're right about that, since there's "similar" notifications used in some places in the Firefox UI (but please see below).

> and people generally don't know about/don't notice tooltips (as we have seen before with questions about outline tree toggling).

Even though it's probably accurate that most people don't care about tooltips, wasn't the case with the outline you refer to somewhat opposite?
Provided that it's issue #7713 and PR #7717 you're thinking of here: the issue (as I understood it) was that there were not tooltip explaining the behavior, which thus necessitated that PR.

One of the reasons that I suggested a new tooltip here, is to hopefully avoid future followups once someone complains that the sidebar notification isn't clear enough as is.